src/lib.rs

```diff
@@ -122,6 +122,77 @@ mod unit;
 mod utils;
 mod value;
 
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum OutputStyle {
+    Expanded,
+    Compressed,
+}
+
+#[derive(Debug)]
+pub struct Options<'a> {
+    pub style: OutputStyle,
+    pub load_paths: Vec<&'a Path>,
+    pub allows_charset: bool,
+    pub unicode_error_messages: bool,
+    pub quiet: bool,
+}
+
+///
+/// `load_paths` - list of paths/files to check for imports for more information see the docs:
+/// - <https://sass-lang.com/documentation/at-rules/import#finding-the-file>
+/// - <https://sass-lang.com/documentation/at-rules/import#load-paths>
+impl<'a> Default for Options<'a> {
+    fn default() -> Self {
+        Self {
+            style: OutputStyle::Expanded,
+            load_paths: Vec::new(),
+            allows_charset: true,
+            unicode_error_messages: true,
+            quiet: false,
+        }
+    }
+}
+
+impl<'a> Options<'a> {
+    /// `grass` currently offers 2 different output styles
+    ///
+    ///  - `OutputStyle::Expanded` writes each selector and declaration on its own line.
+    ///  - `OutputStyle::Compressed` removes as many extra characters as possible, and writes the entire stylesheet on a single line.
+    ///
+    /// By default, output is expanded.
```

Comment:
What do you want to do with these clippy linter errors?

Reply:
Adding the `#[inline]` and `#[must_use]` attributes I suppose makes sense. I think `missing_const_for_fn` might be a false positive here, in which case feel free to add `#[allow(clippy::missing_const_for_fn)]`.

So it might look a bit like
```rust
#[allow(clippy::missing_const_for_fn)]
impl<'a> Options<'a> {
    #[must_use]
    #[inline]
    fn foo() -> Self { ... }
}
```
though the allow should only be used if it gives an error when changing to `const fn`.

Other than that though everything looks fine! It should be ready to merge once CI is passing.